lib/jarbs.rb

```diff
@@ -151,6 +156,32 @@ def run
         end
       end
 
+      command :logging do |c|
+        c.syntax = 'jarbs logging NAME [enable|disable]'
+        c.summary = <<-DESC
+Enable CloudWatch to ElasticSearch logging. You will need to have a lambda available to push logs
```

Comment:
It looks like we already have two lambdas setup to do this.  Should we pick one and default to that?